tests/README.md

```diff
@@ -10,3 +10,14 @@ The test currently builds and tests three versions of the server:
 - Cloudflare worker without authentication
 
 (TODO: Cloudflare worker with authentication)
+
+# S3 Config
+
+If _ALL FIVE_ of the following env vars are set, tests
+with S3 as the storage backend will also be run.
+
+- YSERVE_S3_ACCESS_KEY_ID
```

Comment:
we should use `Y_SWEET_` as the prefix rather than `YSERVE` for consistency.

I'm also ok with no prefix, since AWS credentials are typically passed through [well-known environment variables](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html).

```bash
$ export AWS_ACCESS_KEY_ID=AKIAIOSFODNN7EXAMPLE
$ export AWS_SECRET_ACCESS_KEY=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
$ export AWS_DEFAULT_REGION=us-west-2
```

Reply:
That's true, but that makes it likely to send in access keys and such that are not the ones you intend.

Reply:
I think Y_SWEET as prefix is sensible

Reply:
ah, I see, I misread this as being part of worker code not tests.